assests/styleSheets/general.css

```diff
@@ -0,0 +1,10 @@
+body {
+  margin: 0;
+}
+
+p,
+h1,
+h2,
+span {
+  font-family: var(--primaryFont);
```

Comment:
why just for these elements? it's better to add it to the body.

Reply:
you are right , I added it to `body`